pnpm-lock.yaml

```diff
@@ -4127,11 +4130,11 @@ packages:
     resolution: {integrity: sha512-q+epW0trjVUUHboliPb4UF9g2msf+w61b32tAkFEwL/IwP0DQWgbCMM0Hbe3e3WXSKz5VcUXbzJQgy8Hkra/Lg==}
     dependencies:
       tunnel: 0.0.6
-      undici: 5.28.2
+      undici: 5.27.2
     dev: false
 
-  /@adobe/css-tools@4.3.2:
-    resolution: {integrity: sha512-DA5a1C0gD/pLOvhv33YMrbf2FK3oUzwNl9oOJqE4XVjuEtt6XIakRcsd7eLiOSPkp1kTRQGICTA8cKra/vFbjw==}
+  /@adobe/css-tools@4.3.1:
```

Comment:
It's unfortunate this was downgraded in this PR, as version 4.3.2 includes a security fix. See https://github.com/Automattic/jetpack/security/dependabot/65

Reply:
I'm sure it happened due to a merge conflict and regenerating the lockfile.